modules/using-wildcard-routes.adoc

```diff
@@ -0,0 +1,317 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-controller
+
+[id="using-wildcard-routes_{context}"]
```

Comment:
This module was brought forward directly from 3.11 so the commands and likely the examples are out of date.